packages/block-editor/src/components/block-preview/index.js

```diff
@@ -63,3 +69,57 @@ export function BlockPreview( {
  * @return {WPComponent} The component to be rendered.
  */
 export default memo( BlockPreview );
+
+/**
+ * This hook is used to lightly mark an element as a block preview wrapper
+ * element. Call this hook and pass the returned props to the element to mark as
+ * a block preview wrapper, automatically rendering inner blocks as children. If
+ * you define a ref for the element, it is important to pass the ref to this
+ * hook, which the hook in turn will pass to the component through the props it
+ * returns. Optionally, you can also pass any other props through this hook, and
+ * they will be merged and returned.
+ *
+ * @param {Object}    props                        Optional. Props to pass to the element. Must contain
```

Comment:
Given that `props` are optional, they can be moved to options to remove the need to pass an empty object when someone wants only to set options.

Reply:
Good suggestion, thanks — I've moved `props` to be inside `options` so this function now only has one param 👍